packages/components/src/buttons/src/ClearButton.tsx

```diff
@@ -0,0 +1,104 @@
+import { DismissIcon } from "@hopper-ui/icons";
+import {
+    type StyledComponentProps,
+    useStyledSystem
+} from "@hopper-ui/styled-system";
+import { type ForwardedRef, forwardRef } from "react";
+import {
+    Button as RACButton,
+    type ButtonProps as RACButtonProps,
+    composeRenderProps,
+    useContextProps
+} from "react-aria-components";
+
+import { useLocalizedString } from "../../i18n/index.ts";
+import { Spinner } from "../../Spinner/index.ts";
+import {
+    composeClassnameRenderProps,
+    cssModule
+} from "../../utils/index.ts";
+
+import { ClearButtonContext } from "./ClearButtonContext.ts";
+
+import styles from "./ClearButton.module.css";
+
+export const GlobalClearButtonCssSelector = "hop-ClearButton";
+
+export interface ClearButtonProps extends StyledComponentProps<Omit<RACButtonProps, "children">> {
+
+    /** A clearButton can show a loading indicator.*/
+    isLoading?: boolean;
+}
+
+function ClearButton(props: ClearButtonProps, ref: ForwardedRef<HTMLButtonElement>) {
+    [props, ref] = useContextProps(props, ref, ClearButtonContext);
+
+    const { stylingProps, ...ownProps } = useStyledSystem(props);
+    const stringFormatter = useLocalizedString();
+
+    const {
+        className,
+        isDisabled,
+        isLoading,
```

Comment:
Loading is not necessary in the clear button, you can simplify it

Reply:
it will simplify the css as well

Reply:
I put the loading due to this ticket in Igloo https://github.com/gsoft-inc/ov-igloo-ui/issues/647